room/compiler-processing-testing/src/main/java/androidx/room/compiler/processing/util/runner/JavacCompilationTestRunner.kt

```diff
@@ -33,7 +33,10 @@ internal object JavacCompilationTestRunner : CompilationTestRunner {
     }
 
     override fun compile(params: TestCompilationParameters): CompilationResult {
-        val syntheticJavacProcessor = SyntheticJavacProcessor(params.handlers)
+        val syntheticJavacProcessor = SyntheticJavacProcessor(
+            params.handlers,
+            params.targetLanguage
+        )
```

Comment:
I wonder if it wouldn't be some kind of invalid scenario to have Javac targeting kotlin. The `runJavaProcessorTest()` is not supposed to run with kotlin source for what I can tell.

Reply:
yea this never runs with kotlin sources (in that case, we use kapt instead of javac).
We can make the processing env throw if language is not set to java though that would make testing a bit harder here :/ i'm fine to leave it as is for now.

Reply:
I think I have an idea of how to deal with it here. Will be changing that along with @danysantiago's suggestion.